assessment/models.py

```diff
@@ -462,6 +461,26 @@ class TestFlowAttempt(models.Model):
     event_participation = models.ForeignKey('assessment.AssessmentEventParticipation', on_delete=models.CASCADE)
     test_flow_attempted = models.ForeignKey('assessment.TestFlow', on_delete=models.RESTRICT)
 
+class ResponseTest(AssessmentTool):
+    sender = models.ForeignKey('users.Assessor', on_delete=models.CASCADE)
```

Comment:
Hello, Arya. I believe that the response test sender need not be a foreign key to the Assessor. Rather, it can just be a CharField where the assessor, as the creator, is allowed to set the sender into anyone they think fits the scenario of the assessment.